notebooks/openvino_inference.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>The colon after the last sentence gives the false impression that the next code snippet provides an example, but it does not seem to do that. Perhaps this is still left over from a previous iteration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Replaced colon with the full stop.</p>